### ISSUE TITLE
add sire counter

### ISSUE THREAD

**Issue by g-k-m** [Zero-to-Heroes/firestone#183]
use theotar
enemy has sire at x dmg
from now on u know exactly how much dmg their sire can do at any given time, so pls put firestone tracker for it in case we see their sire

**Comment by g-k-m** [comment]
oh and same if they draw it from moonlit guidance and play 1, and same for devourer keeping track of the infuses can be crucial sometimes

**Comment by sebastientromp** [comment]
Thanks, this indeed seems like something that could be useful to have. It might be a bit complex to develop, so not sure how quickly I can get this out, but I'll give it a try.

**Comment by g-k-m** [comment]
yeah the sire should have a counter like mordresh that increases by 1 for each minion you see dying, while the devourer should show blue in their hand with the blue pic and everything and should dynamically change the infuse amount shown depending on how many minions died, and then when it transforms into red its icon should change to the red one without u having to hover over it so u can easily see at a glance, i don't think a special counter is needed for a revealed devourer, like it is for sire

**Comment by sebastientromp** [comment]
I have something on my list around those lines (basically have an indicator below the turn at which each card was drawn that tells you how much each card was infused). Not sure about the priority though.
